### DISCUSSION TITLE
Issues when using traefik

### DISCUSSION THREAD

**Discussion by johabk** [kimai/kimai#2696] [Questions & Answers]
Can someone cexplain, which stupid error I made in my compose file?

`version: '3.5'
services:

  sqldb:
    image: mysql:5.7
    environment:
      - MYSQL_DATABASE=kimai
      - MYSQL_USER=kimaiuser
      - MYSQL_PASSWORD=kimaipassword
      - MYSQL_ROOT_PASSWORD=changemeplease
    command: --default-storage-engine innodb
    restart: unless-stopped
    networks:
      - kimai
  nginx:
    image: tobybatch/nginx-fpm-reverse-proxy
    ports:
      - "9190:80"
    volumes:
      - public:/opt/kimai/public:ro
    restart: unless-stopped
    depends_on:
      - kimai
    networks:
        - kimaiproxy
        - kimai
    labels:
        - "traefik.enable=true"
        - "traefik.docker.network=proxy_default"
        - "traefik.http.services.kimai-loadbalancer.loadbalancer.server.port=9190"
        - "traefik.http.routers.kimai.rule=Host(`time.mydomain.de`) "
        - "traefik.http.routers.kimai.entrypoints=websecure"
        - "traefik.http.routers.kimai.tls.certresolver=myhttpchallenge"
        - "traefik.http.routers.kimai-unsecure.rule=Host(`time.mydomain.de`)"
        - "traefik.http.routers.kimai-unsecure.entrypoints=web"
        - "traefik.http.routers.kimai-unsecure.middlewares=kimai-redis"
        - "traefik.http.middlewares.kimai-redis.redirectscheme.scheme=https"
  kimai: # This is the latest FPM image of kimai
    image: kimai/kimai2:latest
    environment:
      - ADMINMAIL=admin@kimai.local
      - ADMINPASS=changemeplease
      - DATABASE_URL=mysql://kimaiuser:kimaipassword@sqldb/kimai
      - TRUSTED_HOSTS=nginx,localhost,127.0.0.1,192.168.178.202,time.mydomain.de,172.29.24.8
    networks:
      - kimai
    volumes:
      - public:/opt/kimai/public
      # - var:/opt/kimai/var
      # - ./ldap.conf:/etc/openldap/ldap.conf:z
      # - ./ROOT-CA.pem:/etc/ssl/certs/ROOT-CA.pem:z
    restart: unless-stopped

volumes:
    var:
    public:
networks:
        kimaiproxy:
          external:
            name: proxy_default
        kimai:
          driver: bridge
`

PS: I hope I have posted the question in the right area

**Comment by kevinpapst** [comment]
You might want to post this question in the docker repository https://github.com/tobybatch/kimai2

**Reply by johabk** [reply]
> You might want to post this question in the docker repository https://github.com/tobybatch/kimai2

Yep sounds probably better
